Aptoide/aptoide-client-v8
# [feature/ASV-1562] Ads User Property

**What does this PR do?**

   Adds user property for MoPub visibility status. It essentially tracks the APPC wallet installation when the app first starts and modifies the user property accordingly. Because of some dependency issues, async injection was implemented. Additionally, a new method in MoPubAdsManager was created to streamline its usage.

**Database changed?**

    No

**How should this be manually tested?**

  Check if the user property is being correctly set and also check if the modified events are still correct.

**What are the relevant tickets?**

  [ASV-1562](https://aptoide.atlassian.net/browse/ASV-1562)

**Code Review Checklist**
